core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -889,8 +889,13 @@ protected function generateGlobalButtons()
 			}
 
 			$v = \is_array($v) ? $v : array($v);
-			$label = \is_array($v['label']) ? $v['label'][0] : $v['label'];
-			$title = \is_array($v['label']) ? $v['label'][1] : $v['label'];
+			$title = $label = $k;
```

Comment:
This is a behavior change. Label and title used to be `null` if `$v['label']` was not set.

Reply:
https://github.com/contao/contao/pull/3792/files/8346b5c7e7479fc04c4d3edcc8b601a81046c40a#diff-03666a52b2ba705242aa20dd08b23a30db45719f48607261eabdb45d2cec3b36R915-R923 😉

Reply:
https://github.com/contao/contao/blob/8346b5c7e7479fc04c4d3edcc8b601a81046c40a/core-bundle/src/Resources/contao/classes/DataContainer.php#L915-L923 😉